### PR TITLE
align IDE plugin parameters

### LINKED ISSUE #315: Unify naming of plugin-parameters
ATM some of the plugin-parameters are using an URL-suffix, some do not. Some use uppercased "URL", some use camelCased "Url". It would be nice to unify them. Preferably I would like to drop the URL-suffix where no full-url is needed.

Even if it's a minor-detail, I just stumbled upon this inconsistency :)

I can prepare a PR but wanted to ask first as I'm unsure about the handling of BC-concerns at this point in the project.

### PR DESCRIPTION
**Pull request by benbender**
closes #315

### CODE REVIEW

File: packages/plugin/altair/src/integrations/cloudflare.ts [hunk 1]
```diff
@@ -11,9 +11,10 @@ export const handleCloudflare: IntegrationRegisterHandler<'cloudflare'> = async
 
   const render = await ctx.altair.render;
 
-  const { endpointURL = ctx.options.path || '/graphql', baseURL: baseURLOpt, path: _path, ...renderOptions } = ctx.altair.options;
+  const { endpoint = ctx.options.path || '/graphql', base: baseURLOpt, path: _path, ...renderOptions } = ctx.altair.options;
 
   const baseURL = withTrailingSlash(baseURLOpt || path);
+  const endpointURL = endpoint;
```
Comment (PabloSzx) [inline packages/plugin/altair/src/integrations/cloudflare.ts:17]:
```suggestion
  const { endpoint: endpointURL = ctx.options.path || '/graphql', base: baseURLOpt, path: _path, ...renderOptions } = ctx.altair.options;

  const baseURL = withTrailingSlash(baseURLOpt || path);
```

File: packages/plugin/altair/src/integrations/sveltekit.ts [hunk 2]
```diff
@@ -8,17 +8,13 @@ export const handleSvelteKit: IntegrationRegisterHandler<'sveltekit'> = async ({
...
-    path: _path,
-    ...renderOptions
-  } = ctx.altair.options;
+  const { endpoint = ctx.options.path || '/api/graphql', base: baseURLOpt, path: _path, ...renderOptions } = ctx.altair.options;
 
   const baseURL = withTrailingSlash(baseURLOpt || path);
 
   const baseURLTrailing = withTrailingSlash(ctx.altair.baseURL);
   const baseURLNoTrailing = withoutTrailingSlash(ctx.altair.baseURL);
+  const endpointURL = endpoint;
```
Comment (PabloSzx) [inline packages/plugin/altair/src/integrations/sveltekit.ts:17]:
```suggestion
  const { endpoint: endpointURL = ctx.options.path || '/api/graphql', base: baseURLOpt, path: _path, ...renderOptions } = ctx.altair.options;

  const baseURL = withTrailingSlash(baseURLOpt || path);

  const baseURLTrailing = withTrailingSlash(ctx.altair.baseURL);
  const baseURLNoTrailing = withoutTrailingSlash(ctx.altair.baseURL);
```

File: packages/plugin/altair/src/types.ts [hunk 3]
```diff
@@ -2,11 +2,21 @@ import type { PickRequired } from 'graphql-ez';
...
+export interface AltairOptions extends Omit<RenderOptions, 'baseURL' | 'endpointURL'> {
   /**
    * @default "/altair"
    */
   path?: string;
+
+  /**
+   * URL to be used as a base for relative URLs
+   */
+  base?: string;
```
Comment (PabloSzx) [inline packages/plugin/altair/src/types.ts:14]: I feel like this specific change is not needed at all, baseURL is altair specific and doesn't make sense to remove the "URL" just for no real reason

**Review by PabloSzx — CHANGES_REQUESTED** [review]
Can you create a new changeset like this?

```md
---
'@graphql-ez/plugin-altair': minor
'@graphql-ez/plugin-voyager': minor
---

Rename "endpointURL" and "endpointUrl" to "endpoint" to align plugins configuration
```

File: packages/plugin/altair/src/types.ts (hunk 3 above)
Comment (benbender) [reply packages/plugin/altair/src/types.ts:14]: I was unsure about both of them, but opted for the unifying approach as both are part of the user-facing interface. I would say it's a matter of taste if one opts for the consistency with graphql-ez or with altair. You only want  to have`base`, but not `endpoint`, removed, right?

Comment (PabloSzx) [reply packages/plugin/altair/src/types.ts:14]: baseURL doesn't ever need to be manually specified, just specifying "path" is enough, baseURL doesn't need to be highlighted

Comment (PabloSzx) [reply packages/plugin/altair/src/types.ts:14]: yeah, only base, aligning endpoint is valuable

**Review by PabloSzx — APPROVED** [review]

**Pull request merged**